Mantis.xcodeproj/project.pbxproj

```diff
@@ -43,7 +45,6 @@
 		OBJ_80 /* CropToolbar.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_24 /* CropToolbar.swift */; };
 		OBJ_81 /* CropViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_25 /* CropViewController.swift */; };
 		OBJ_82 /* FixedRatioManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_26 /* FixedRatioManager.swift */; };
-		OBJ_83 /* RatioPresenter.swift in Sources */ = {isa = PBXBuildFile; fileRef = OBJ_27 /* RatioPresenter.swift */; };
```

Comment:
We should keep RatioPresenter because other users may still need it.

Reply:
My purpose is to replace RatioPresenter with RatioSelector, so I think it is unnecessary